portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -82,4 +88,33 @@ private String getParameter(HttpServletRequest request, String name, String defa
     }
     return value;
   }
+
+  /**
+   * @return the number of comments to display, or -1 if all comments should be displayed
+   */
+  private int getNumCommentsToDisplay(HttpServletRequest request) {
+    // Get the input from the form.
+    String numCommentsString = getParameter(request, "num-comments", null);
+
+    if (numCommentsString.equals("All")) {
```

Comment:
This if fine for this project, but generally I would consider always having some limit :) If this gets to be too many comments, I might consider pagination.